tracing/tracing.go

```diff
@@ -146,3 +152,30 @@ func LogKV(k, v string, ctx context.Context) {
 		span.LogKV(k, v)
 	}
 }
+
+// GetTraceID retrieves TraceID from HTTP request, for example to search for this trace
+// in the UI of your tracing solution and to get more context about it
+func GetTraceID(req *http.Request) string {
```

Comment:
Lets pass it `ot.Span` instead of request to not create a dependency on net/http